### PR TITLE
KDE Q4 report

### PR DESCRIPTION
**Pull request by adriaandegroot**

### CODE REVIEW

File: 2021q4/kde.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Elsewhere in the software stack, kde@ also maintains ports that support the desktop in general. Some highlights are:
+
+* *devel/libphonenumber* keeps chasing changes to the world's phone numbers (the FreeBSD foundation can be reached at +1.720.207.5142).
+* *graphics/poppler* updated this much-used PDF-rendering library.
+* *multimedia/pipewire*, the audio-and-video successor to pulseaudio, was updated and now supports SSL as well.
+* *net/py-pytradfri* got several updates so you can control your lights from FreeBSD.
+* *print/freetype2* was updated to the latest release; relatedly, there was am update to *x11-toolkits/libXft*.
+* *print/harfbuzz*, the text-shaping library, was updated for more font type support.
+* *sysutils/bsdisks* is an implementation of DBus interfaces for examining disks (drives, partitions, etc.). It is also used for removable-disk notifications.
```
Comment (pauamma) [inline 2021q4/kde.adoc:37]: https://cgit.freebsd.org/ports/tree/sysutils/bsdisks/Makefile#n5 says otherwise. (Or did you mean "people part of kde@"?)

**Review by pauamma — COMMENTED** [review]
Other than clearing that possible mistake, LGTM.

**Comment by adriaandegroot** [comment]
On Wednesday, 29 December 2021 03:35:38 CET Pau Amma wrote:
> https://cgit.freebsd.org/ports/tree/sysutils/bsdisks/Makefile#n5 says
> otherwise. (Or did you mean "people part of kde@"?)

That, yes. If it doesn't get mentioned elsewhere, e.g. in desktop@, it should 
be mentioned here. 

[ade]

**Comment by pauamma** [comment]
> On Wednesday, 29 December 2021 03:35:38 CET Pau Amma wrote: https://cgit.freebsd.org/ports/tree/sysutils/bsdisks/Makefile#n5 says otherwise. (Or did you mean "people part of kde@"?)
> That, yes. If it doesn't get mentioned elsewhere, e.g. in desktop@, it should be mentioned here. [ade]

Fair enough. If some other group mentions it, we can just merge them and move them somewhere.

**Review by pauamma — APPROVED** [review]

**Review by pauamma — APPROVED** [review]

**Pull request merged**